tools/@o3r/build-helpers/scripts/prepare-publish.mjs

```diff
@@ -57,3 +57,4 @@ fields.forEach((field) => {
 
 writeFileSync(distPackageJson, JSON.stringify(packageJson, null, 2));
 copyFileSync(resolve(dirname(privatePackageJson.path), 'LICENSE'), resolve(distPath, 'LICENSE'));
+copyFileSync(resolve(dirname(privatePackageJson.path), 'README.md'), resolve(distPath, 'README.md'));
```

Comment:
Are you sure you want to copy the repository readme into all packages' dist folder?
The purpose would not to have 1 readme file per package instead?